pkg/testsuite/testsuite.go

```diff
@@ -461,7 +461,7 @@ func (t *TestSuite) DefaultWallet() *mock.Wallet {
 }
 
 func (t *TestSuite) AddWallet(name string, node *mock.Node, accountID iotago.AccountID, keyManager ...*wallet.KeyManager) *mock.Wallet {
-	newWallet := mock.NewWallet(t.Testing, name, &mock.TestSuiteClient{Node: node}, keyManager...)
+	newWallet := mock.NewWallet(t.Testing, name, &mock.TestSuiteClient{Node: node}, &mock.TestSuiteWalletClock{}, keyManager...)
```

Comment:
What about having `mock.NewTestSuiteClient(node)` and `mock.NewTestSuiteWalletClock()` just fora anicer look